web/src/hooks/getEstimationTranslation.ts

```diff
@@ -0,0 +1,21 @@
+import { useTranslation } from 'react-i18next';
+
+export function useGetEstimationTranslation(
+  field: 'title' | 'pill' | 'body',
+  estimationMethod?: string,
```

Comment:
Since we are using Enums now this should accept the enum only.

Reply:
Done! ...and in many other places 😎